### PR TITLE
Sample TCK runner with Glassfish moved to bundle and Userguide changes

### PR DESCRIPTION
**Pull request by alwin-joseph**

### CODE REVIEW

File: tck/docs/userguide/src/main/jbake/content/attributes.conf [hunk 1]
```diff
@@ -23,15 +23,15 @@
 :TechnologyHomeEnv: CAJ_HOME
 // Java SE version required.
 :SEversion: 17
```
Comment (lukasj) [inline tck/docs/userguide/src/main/jbake/content/attributes.conf:25]: SEversion should probably be 17+ (or 17 and 21?)

**Review by lukasj — COMMENTED** [review]
* ca.sig... properties are no longer supported, aren't they? If so, they should go away from the UG 
* a TCK distribution should probably also contain a `TCK-Exclude-List.txt` file, since it is being referred from the UG; the file can be empty with a comment "No tests are excluded in this version." (or sth like that)

**Comment by alwin-joseph** [comment]
> * ca.sig... properties are no longer supported, aren't they? If so, they should go away from the UG
> 
IIUC jakarta.annotation.ManagedBean does not exist anymore. Hence ca.sig.managedbean propertly is removed. All other ca.sig.* properties are supported.

>     * a TCK distribution should probably also contain a `TCK-Exclude-List.txt` file, since it is being referred from the UG; the file can be empty with a comment "No tests are excluded in this version." (or sth like that)

The file exists at docs folder.

**Review by lukasj — APPROVED** [review]
thanks!

**Pull request merged**